cypress/cypress/component/Input/BasicInputField.cy.tsx

```diff
@@ -95,6 +95,37 @@ describe("Input.cy.tsx", () => {
     cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: randomWord });
   });
 
+  it("validation works for sub-objects", () => {
+    const objectName = faker.random.alpha(10);
+    let name = faker.random.alpha(10);
+    const errorMessage = faker.random.words();
+    const schema = yup.object().shape({
+      [objectName]: yup.object().shape({
+        [name]: yup.string().required(errorMessage),
+      }),
+    });
+
+    name = `${objectName}.${name}`;
```

Comment:
Could you please create a new const `fieldName` here.
We generally try to avoid using let because the code becomes more difficult to read because you have to manually check if it gets changed somwhere.

Reply:
yes sure, i just saw now that you already proposed a name, i hope the one i used is also okay.. sorry.